src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const baseCost = 40;
+  const discountFor3Days = 20;
+  const discountFor7Days = 50;
+  const mediumTerm = 3;
+  const longTerm = 7;
+
+  if (days >= longTerm) {
+    return baseCost * days - discountFor7Days;
+  } else if (days >= mediumTerm) {
+    return baseCost * days - discountFor3Days;
+  }
```

Comment:
No need to use `else if` here - as long as you have returns from inside the blocks you may keep it more simple:

```suggestion
  if (days >= longTerm) {
    return baseCost * days - discountFor7Days;
  } 
  
  if (days >= mediumTerm) {
    return baseCost * days - discountFor3Days;
  }
```